tests/stubtest_third_party.py

```diff
@@ -109,19 +112,35 @@ def run_stubtest(dist: Path) -> bool:
     return True
 
 
-def run_stubtest_for(metadata: dict[str, Any], dist: Path) -> bool:
-    return has_py3_stubs(dist) and metadata.get("stubtest", True)
-
-
 # Keep this in sync with mypy_test.py
 def has_py3_stubs(dist: Path) -> bool:
     return len(glob(f"{dist}/*.pyi")) > 0 or len(glob(f"{dist}/[!@]*/__init__.pyi")) > 0
 
 
+def install_apt_packages(dist: Path, metadata: dict[str, Any], install: bool) -> bool:
+    apt_packages = metadata.get("stubtest_apt_dependencies", [])
+    if not apt_packages:
+        return True
+    if not install:
+        print(f"Ensure the following packages are installed for {dist.name}: {', '.join(apt_packages)}", file=sys.stderr)
+        return True
+    try:
+        apt_cmd = ["apt", "install", *apt_packages]
```

Comment:
Does this really work without `sudo`?

Reply:
Apparently not https://github.com/python/typeshed/runs/4636602024?check_suite_focus=true